tsc/meetings/2019-08-29.md

```diff
@@ -0,0 +1,130 @@
+Minutes from 26th OpenVDB TSC meeting, July 25, 2019
```

Comment:
wait what, July 25?!? Isn't this minutes for last week's meeting on August 29/30? Since I am regularly moving between totally different time-zone I'd like for us to start indicating time-zones when we talked about dates and times (e.g. GMT). Technically our last meeting was on August 30 for me :)

Reply:
My secret of always starting from the last meeting notes is exposed!  I was so busy trying to get the meeting number right (because the 25th meeting hadn't been merged yet) I didn't notice I didn't change the date.  The filename at least I got right.

Not sure what to do about timezone there as there is no time, so adding (GMT) to the end is a bit odd.